### PR TITLE
fix: SgtPooki can manage issue labels in js-libp2p-examples

### PR DESCRIPTION
**Pull request by SgtPooki**
### Summary

Grant SgtPooki admin access to js-libp2p-examples

### Why do you need this?

So I can add/adjust labels in the example repo: https://github.com/libp2p/js-libp2p-examples

### What else do we need to know?

I don't need full Admin, but triage rights would allow me to help monitor/maintain repos I can help with

**DRI:** myself

### Reviewer's Checklist

- [ ] It is clear where the request is coming from (if unsure, ask)
- [ ] All the automated checks passed
- [ ] The YAML changes reflect the summary of the request
- [ ] The Terraform plan posted as a comment reflects the summary of the request

### CODE REVIEW

File: github/libp2p.yml [hunk 1]
```diff
@@ -3413,6 +3490,7 @@ repositories:
     teams:
       admin:
         - Admin
+        - SgtPooki
```
Comment (SgtPooki) [inline github/libp2p.yml:3493]: should I be in `maintain`, or should we add team `shipyard`?

File: github/libp2p.yml [hunk 2]
```diff
@@ -3381,6 +3451,13 @@ repositories:
     default_branch: main
     delete_branch_on_merge: false
     description: Connect a browser to another browser
+    files:
+      .github/pull_request_template.md:
+        content: .github/js_pull_request_template.md
+      .github/workflows/semantic-pull-request.yml:
+        content: .github/workflows/semantic-pull-request.yml
+      .github/workflows/stale.yml:
+        content: .github/workflows/stale.yml
```
Comment (SgtPooki) [inline github/libp2p.yml:3460]: these changes were not added by me.. I created PR through github.. I can remove if needed

**Review by SgtPooki — COMMENTED** [review]
self review

File: github/libp2p.yml [hunk 3]
```diff
@@ -3490,6 +3490,7 @@ repositories:
     teams:
       admin:
         - Admin
+        - SgtPooki
         - w3dt-stewards
```
[the remark this answers is not in the document]
Comment (galargh) [reply github/libp2p.yml:3494]: Yes, exactly this 👍

File: github/libp2p.yml [hunk 4]
```diff
@@ -3490,7 +3490,10 @@ repositories:
     teams:
       admin:
         - Admin
+        - SgtPooki
```
Comment (p-shahi) [inline github/libp2p.yml:3493]:
```suggestion
```

**Review by p-shahi — APPROVED** [review]

**Pull request merged**